docker-compose.yml

```diff
@@ -101,25 +101,28 @@ services:
     command: ["workspace", "query-node-root", "processor:start"]
 
   indexer:
-    image: joystream/apps
+    image: joystream/hydra-indexer:2.0.1-beta-v3.9
     restart: unless-stopped
-    build: 
-      context: .
-      dockerfile: apps.Dockerfile
     env_file:
       # relative to working directory where docker-compose was run from
       - .env
     environment:
       - TYPEORM_HOST=db
       - TYPEORM_DATABASE=${INDEXER_DB_NAME}
       - INDEXER_WORKERS=5
-      - PROCESSOR_POLL_INTERVAL=1000 # refresh every second 
+      - PROCESSOR_POLL_INTERVAL=1000 # refresh every second
       - REDIS_URI=redis://redis:6379/0
       - DEBUG=index-builder:*
       - WS_PROVIDER_ENDPOINT_URI=ws://joystream-node:9944
-    depends_on: 
+      - TYPES_JSON=types.json
+      - DB_NAME=${INDEXER_DB_NAME}
+      - DB_HOST=db
+    depends_on:
       - db
-    command: ["workspace", "query-node-root", "indexer:start"] 
+    volumes:
+      - ./types/augment/all/defs.json:/home/hydra/packages/hydra-indexer/types.json
+    command: >
+      sh -c 'yarn db:bootstrap && yarn start:prod'
```

Comment:
I don't see `db:bootstrap` in any package.json ?

Reply:
or is that in indexer container ?

Reply:
yes, correct! it's in the indexer.